plugins/filter/junit2obj.py

```diff
@@ -0,0 +1,287 @@
+# Copyright (C) 2024 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+__version__: str = "1.0.0"
+
+import json
+import time
+
+try:
+    from typing import Any  # for Python 3.8++
+except ImportError:
+    from typing_extensions import Any  # type: ignore  # for older Python versions
+from typing import Dict  # type: ignore
```

Comment:
Hi @mvk could you please avoid modifying the Python import globally? I believe these lines cause the other filters to fail by polluting global import namespace and can be safely removed.